build/notarize.py

```diff
@@ -0,0 +1,99 @@
+import os
+import subprocess
+import tqdm
+
+ZIPNAME = 'cmu_graphics_installer.zip'
+TEAM_ID = 'LXH25PRRZ2'
```

Comment:
@schmave Since this identifier is visible to anybody who runs `codesign -dv` on our binaries, it seemed ok to include directly in the script